syncthing/STApplication.m

```diff
@@ -163,7 +163,13 @@ - (IBAction) clickedOpen:(id)sender {
 - (void) updateFoldersMenu:(NSMenu *)menu {
     [menu removeAllItems];
     
-    for (id dir in [self.syncthing getFolders]) {
+    // Get folders from syncthing and sort ascending
+    NSSortDescriptor *sort = [NSSortDescriptor sortDescriptorWithKey:@"label" ascending:YES comparator:^NSComparisonResult(id obj1, id obj2) {
```

Comment:
It should work without a comparator. NSSortDescriptor defaults to `compare:` anyway.